person.rb

```diff
@@ -1,13 +1,8 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :rentals
 
-class Person < Nameable
-  def initialize(age, name)
-    super()
-    @age = age
+  def initialize(name)
```

Comment:
- [ ] Please add the `age`, and `parent_permission` set to true in the constructor's parameters.